arch/power/clang_workaround.h

```diff
@@ -0,0 +1,98 @@
+/* Helper functions to work around issues with clang builtins
```

Comment:
There is a `fallback_builtins.h` in the main directory. I think it should be moved to arch/x86/fallback_builtins.h. And then this file should be renamed arch/power/fallback_builtins.h.

Reply:
I renamed `clang_workarounds.h` to `fallback_builtins.h`. I haven't moved the x86 one though.